packages/jest-runner/src/run_test.js

```diff
@@ -90,7 +98,7 @@ async function runTestInternal(
   } else if (globalConfig.verbose) {
     testConsole = new Console(consoleOut, process.stderr, consoleFormatter);
   } else {
-    testConsole = new BufferedConsole();
+    testConsole = new BufferedConsole(() => runtime && runtime.getSourceMaps());
```

Comment:
went with a getter as `consoleFormatter` seemed like it did too much

Reply:
looks great

fwiw i don't think there's an integration test for this path